packages/accordion/package.json

```diff
@@ -17,7 +17,7 @@
     ],
     "license": "MIT",
     "scripts": {
-        "clean": "rimraf build/ dist/ .cache node_modules/",
+        "clean": "rimraf **/*.css",
```

Comment:
Isn't parcel in use in most style pkgs? In that case, we should keep the `dist/` remove

Reply:
Good point! I'll add it back

Reply:
`node_modules/` should probably also still be in there 🤔